src/components/Dropdown/dropdown.module.scss

```diff
@@ -0,0 +1,60 @@
+@import '@/styles/mediaQueries.scss';
+@import '@/styles/variables.scss';
+@import '@/styles/colors.scss';
+
+.dropdown {
+  position: relative;
+  /* display: inline-block; */
+
+  .dropdownBtn {
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+    background-color: $white;
+    border: 1.5px solid $black;
+    border-radius: $brad-sm;
+    padding: 0.5rem;
+    padding-top: 0.75rem;
```

Comment:
This looks odd. I'd rather go for a unified padding and a center alignment via align-items: center